### PR TITLE
Add French language

### PR DESCRIPTION
**Pull request by bmenant**
If someone could review and check it out, that’d be appreciated. 

I’m not quite sure about _pattern/modèle_. It’s hard to understand that one if you have no clue what a _pattern_ means over here. Most of the time, the English word _pattern_ is used in French. But few people actually understand... so I went for the French word _modèle_. A note/link/example would be much appreciated though.

### CODE REVIEW

File: fr/examples/gtd-4.html [hunk 1]
```diff
@@ -0,0 +1,109 @@
+---
+redirect_from:
+- /en/examples/gtd-4.html
```
Comment (jgkim) [inline fr/examples/gtd-4.html]: The line 3 should be `/fr/examples/gtd-4.html`.

File: fr/examples/gtd-4.html [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+- /gtd-4.html
```
Comment (jgkim) [inline fr/examples/gtd-4.html]: The line 4 should be deleted.

File: fr/examples/gtd-5.html [hunk 3]
```diff
@@ -0,0 +1,111 @@
+---
+redirect_from:
+- /en/examples/gtd-5.html
```
Comment (jgkim) [inline fr/examples/gtd-5.html]: The same thing goes here. This should be `/fr/examples/gtd-5.html`.

File: fr/examples/gtd-5.html [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+- /gtd-5.html
```
Comment (jgkim) [inline fr/examples/gtd-5.html]: This should be deleted.

File: fr/index.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+---
+layout: default
+redirect_from: /
```
Comment (jgkim) [inline fr/index.md]: This should be `redirect_from: /fr/`.

**Review by jgkim — CHANGES_REQUESTED** [review]
There are a few redirection errors to be corrected.

**Comment by jgkim** [comment]
@bmenant I cannot read French so I cannot review the content. If you know someone can review the content, please ping him or her here. In the meantime, please fix the errors mentioned above. Thank you for the PR. :)

**Comment by bmenant** [comment]
@jgkim Errors fixed https://github.com/mhausenblas/5stardata.info/pull/29/commits/7ccfaf74e4b59d19759cbf66c46fb09c79e63726

**Review by jgkim — APPROVED** [review]

**Pull request merged**